GVFS/GVFS.Platform.Mac/ProjFSKext.cs

```diff
@@ -73,5 +69,40 @@ public bool TryPrepareFolderForCallbacks(string folderPath, out string error, ou
 
             return true;
         }
+
+        public bool TryLoad(ITracer tracer, string enlistmentRoot, out string errorMessage)
+        {
+            EventMetadata metadata = new EventMetadata();
+            ProcessResult loadKext = ProcessHelper.Run("sudo", "/sbin/kextload -b " + DriverName);
+            if (loadKext.ExitCode == LoadKext_ExitCode_Success)
+            {
+                tracer.RelatedWarning(metadata, $"{DriverName} was successfully loaded.", Keywords.Telemetry);
+                errorMessage = null;
+                return true;
+            }
+            else if (loadKext.ExitCode == LoadKext_ExitCode_NotApproved)
+            {
+                tracer.RelatedError("Kext unable to load. Not approved by the user");
+                errorMessage = DriverName + @" was unable to load.  Please check and make sure you have allowed the extension in
+System Preferences -> Security & Privacy";
+            }
+            else
+            {
+                metadata.Add("ExitCode", loadKext.ExitCode);
+                metadata.Add("Output", loadKext.Output);
+                metadata.Add("Errors", loadKext.Errors);
+                tracer.RelatedError(metadata, "Failed to load kext");
+
+                errorMessage = DriverName + " is not loaded. Make sure the driver is loaded and try again.";

```

Comment:
Error message suggestion: `sure the driver is loaded` -> `sure the kext is loaded`